test/e2e/framework/test_context.go

```diff
@@ -103,6 +103,8 @@ type TestContextType struct {
 	// Node e2e specific test context
 	NodeTestContextType
 
+	// Monitoring solution that is used in current cluster. Values are coming from ENABLE_CLUSTER_MONITORING flag.
```

Comment:
You mean 'KUBE_ENABLE_CLUSTER_MONITORING env var'?
I would skip mentioning that, as it's not necessary that the value has to come from there - someone can also just set the flag.